### PR TITLE
check and remove python cli components

### PR DESCRIPTION
**Pull request by sahil-lakhwani**
Signed-off-by: sahil-lakhwani <sahilakhwani@gmail.com>

Checks if the any of the Python CLI components are present on the machine and removes them.

### CODE REVIEW

**Review by anupbarve — CHANGES_REQUESTED** [review]
Shouldn't these be integrated in check-node and prep-node flow ?

**Comment by anupbarve** [comment]
`checkPyCli` is not called in `Check` function. Am I missing something ?

**Comment by sahil-lakhwani** [comment]
> `checkPyCli` is not called in `Check` function. Am I missing something ?

yeah my bad @anupbarve . Some check-in issue I believe, have pushed a new commit.

File: pkg/platform/centos/centos.go [hunk 1]
```diff
@@ -149,3 +152,24 @@ func (c *CentOS) checkPort() (bool, error) {
 
 	return true, nil
 }
+
+func (c *CentOS) checkPyCli() (bool, error) {
```
Comment (anupbarve) [inline pkg/platform/centos/centos.go:156]: `checkPyCli` seems misnomer here. 
Maybe use `erasePyCli` or `removePyCli`.

**Review by anupbarve — DISMISSED** [review]
nit to be addressed.
Rest looks good.
LGTM

**Review by anupbarve — APPROVED** [review]

**Pull request merged**